go.mod

```diff
@@ -5,12 +5,10 @@ go 1.12
 require (
 	github.com/DataDog/zstd v1.4.0 // indirect
 	github.com/Shopify/sarama v1.23.0
-	github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239 // indirect
```

Comment:
I upgraded to go 1.13 and I did not get any changes in these files can you try upgrading to that?